owncloudApp/src/main/java/com/owncloud/android/operations/CreateFolderOperation.java

```diff
@@ -35,6 +35,8 @@
  * Access to remote operation performing the creation of a new folder in the ownCloud server.
  * Save the new folder in Database
  */
+@Deprecated
```

Comment:
`@Deprecated("Call usecase instead of this operation. Remove as soon as possible")`

Reply:
I cannot add reason in java files 😭

Reply:
:sad: